app/main.py

```diff
@@ -16,8 +32,35 @@ def robots():
         return f.read()
 
 
+@app.post("/analyze_data")
+async def get_evaluation(board_data: EvaluationModel):
+    fen_position = board_data.fen
+    stockfish.set_fen_position(fen_position)
+
+    sf_eval = stockfish.get_static_eval()
+    return {"msg": "Great success!", "evaluation": sf_eval}
+
+
+@app.post("/analyze_move")
+async def analyze_move(board_data: AnalysisModel):
+    if stockfish.is_fen_valid(board_data.fen):
+        stockfish.set_fen_position(board_data.fen)
+    else:
+        return {"error": "invalid fen", "fen": board_data.fen}
+
+    sf_eval_before = stockfish.get_static_eval()
+    stockfish.make_moves_from_current_position([board_data.move])
+    sf_eval_after = stockfish.get_static_eval()
+    eval_diff = sf_eval_after - sf_eval_before
+    return {
+        "msg": "Great success!",
+        "move_evaluation": eval_diff,
+        "evaluation": sf_eval_after,
```

Comment:
Naming, naming, naming... `move_evaluation` vs `evaluation` vs `score` vs `rating` vs `analysis` vs... - what's the difference? think about what the numbers _actually_ mean and call them that